src/chain_spec.rs

```diff
@@ -97,6 +102,10 @@ pub fn get_authority_keys_from_seed(
     (
         get_account_id_from_seed::<sr25519::Public>(&format!("{}//stash", seed)),
         get_account_id_from_seed::<sr25519::Public>(seed),
+        // get_from_seed::<GrandpaId>(&format!("{}//gran", seed)),
+        // get_from_seed::<BabeId>(&format!("{}//babe", seed)),
+        // get_from_seed::<ImOnlineId>(&format!("{}//imon", seed)),
+        // get_from_seed::<AuthorityDiscoveryId>(&format!("{}//auth", seed)),
```

Comment:
Clean up extra comments?

Reply:
I'm going to leave these in for now. One slightly annoying thing is that when `get_authority_keys_from_seed` is called, for e.g. Alice, it uses the same seed for all of the session keys. This makes switching it for the "real" keys a bit of a pain as it isn't easy to differentiate between them. Adding an additional derivation path works for this, but then local nodes based on Alice etc. don't work as the node doesn't include these additional keys.

There are ways to fix this, but not a top priority, so leaving this in, in the short term.